transactions/2.0/_index.md

```diff
@@ -5,19 +5,19 @@ summary: "Release for Jakarta EE 9"
 ---
 Jakarta Transactions - Need scope statement
 
-* [Jakarta Transactions 2.0 Release Record]()
+* [Jakarta Transactions 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jta/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
 * [Jakarta Transactions 2.0 Specification Document]() (PDF)
```

Comment:
If you complete a version of the Spec later, we can always update this _index.md to reference the pdf and html generated Spec.  One suggestion is to name the file with some qualifier like -SNAPSHOT, or -M1, or -RC2.  Just something to indicate it's not Final.  And, the header page within your Spec should indicate it's not Final yet (Draft, or Public Draft, or something to that effect).

Reply:
Thanks, if and when we have an updated version of the spec ready to share I will update this folder with it and the _index.md

Reply:
Sounds good, @tomjenkinson.  Thanks.